docs/source/torch.rst

```diff
@@ -190,14 +188,12 @@ Pointwise Ops
 .. autofunction:: add
 .. autofunction:: addcdiv
 .. autofunction:: addcmul
-.. autofunction:: angle
```

Comment:
Same question as above. Do these autofunctions in torch.rst need to stay? Just flagging cause these aren't part of changes I want to make.

Reply:
yea .. I put them back in..